spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -4,7 +4,7 @@
 #
 # Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
 # Author: Charley
-# Modified: 2017-08-19
+# Modified: 2023-05-16 by Jan Valosek
```

Comment:
Do we want to keep this line? Since we use git/github, is it necessary?

Reply:
not necessary

Reply:
I reset the "Modified" line because all script headers will be cleaned globally in a separate PR (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4117).